<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubemapEditorDialog</class>
 <widget class="QDialog" name="CubemapEditorDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cubemap Editor</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="ClickableQLabel" name="labelPZ">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableQLabel" name="labelNY">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableQLabel" name="labelNZ">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableQLabel" name="labelPY">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableQLabel" name="labelNX">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="ClickableQLabel" name="labelPX">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Click to load face</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>367</y>
     <width>128</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save current texture</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClose">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>403</y>
     <width>128</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close without saving</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>280</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Face texture requirements: square, size is power of 2, all faces have same size and format.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Face width:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Face height:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFaceWidth">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFaceHeight">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example4.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example2.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example5.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example0.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="QtIcons.qrc">:/QtImages/sky_example3.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="legend" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="QtIcons.qrc">:/QtImages/sky_example6.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sky</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ground</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="QtIcons.qrc">:/QtImages/sky_example7.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblSaving">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Saving...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_14</zorder>
  <zorder>label_13</zorder>
  <zorder>label_12</zorder>
  <zorder>label_11</zorder>
  <zorder>label_10</zorder>
  <zorder>labelNZ</zorder>
  <zorder>label_9</zorder>
  <zorder>labelNY</zorder>
  <zorder>labelPY</zorder>
  <zorder>labelNX</zorder>
  <zorder>labelPX</zorder>
  <zorder>buttonSave</zorder>
  <zorder>buttonClose</zorder>
  <zorder>labelDescription</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>labelFaceWidth</zorder>
  <zorder>labelFaceHeight</zorder>
  <zorder>labelPZ</zorder>
  <zorder>legend</zorder>
  <zorder>lblSaving</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableQLabel</class>
   <extends>QLabel</extends>
   <header>ClickableQLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
